enterprise/server-enterprise/src/main/java/org/neo4j/server/enterprise/StandaloneClusterClient.java

```diff
@@ -65,11 +67,13 @@
 public class StandaloneClusterClient
 {
     private final LifeSupport life = new LifeSupport();
+    private final Timer timer;
 
     private StandaloneClusterClient( Logging logging, ClusterClient clusterClient )
     {
         life.add( logging );
         life.add( clusterClient );
+        timer = new Timer();
```

Comment:
Please make the Timer a daemon, so it doesn't prevent normal shutdown by exiting from main().